package.json

```
@@ -75,7 +75,7 @@
     "react-nanny": "^2.15.0",
     "react-native": "0.69.6",
     "react-native-collapsible-tab-view": "^5.0.0-rc.13",
-    "react-native-config": "^1.4.11",
+    "react-native-config": "https://github.com/artsy/react-native-config",
```

```suggestion
    "react-native-config": "https://github.com/artsy/react-native-config.git#v1.4.11-artsy",
```
i suggest we use this, so we have a way to version this